scripts/generate_gift_cards_for_campaign.rb

```diff
@@ -0,0 +1,20 @@
+def max_campaign(payment_intent_id, campaign_id)
```

Comment:
are we passing in the `payment_intent_id` that we generate with the `custom_payment_intent` script?